cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -314,8 +314,16 @@ EOF
 }
 
 function configure_sql_client() {
+  local mysql_conf_dir
+  if [[ -d /etc/mysql/conf.d ]]; then
+    mysql_conf_dir=/etc/mysql/conf.d
+  elif [[ -d /etc/my.cnf.d ]]; then
+    mysql_conf_dir=/etc/my.cnf.d
+  else
+    err "Neither /etc/mysql/conf.d nor /etc/my.cnf.d exists"
+  fi
   # Configure MySQL client to talk to metastore
-  cat <<EOF >/etc/mysql/conf.d/cloud-sql-proxy.cnf
+  cat <<EOF >"${mysql_conf_dir}/cloud-sql-proxy.cnf"
```

Comment:
Please uncomment test on Rocky to make sure that change works: https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/cloud-sql-proxy/test_cloud_sql_proxy.py#L59-L60

Reply:
Done.